File: data/java/TRIANGULAR_NUMBERS_1_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(1);
    param0.add(3);
    param0.add(6);
    param0.add(10);
    param0.add(55);
    param0.add(48);
    param0.add(63);
    param0.add(72);
    param0.add(16);
    param0.add(85);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i)) == f_gold(param0.get(i)))
	        {
                builder.append("TRIANGULAR_NUMBERS_1," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("TRIANGULAR_NUMBERS_1," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("TRIANGULAR_NUMBERS_1," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/TRIANGULAR_NUMBERS_1_MAIN.py
if __name__ == '__main__':
    param = [
    (1,),
    (3,),
    (6,),
    (10,),
    (55,),
    (48,),
    (63,),
    (72,),
    (16,),
    (85,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("TRIANGULAR_NUMBERS_1," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("TRIANGULAR_NUMBERS_1," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
